registrations/utils.py

```diff
@@ -49,3 +50,7 @@ def send_mass_html_mail(
         messages.append(mail)
 
     return connection.send_messages(messages)
+
+
+def get_email_noreply_address():
+    return "noreply@%s" % Site.objects.get_current().domain
```

Comment:
Pretty magical. Would be nice if you can override this through a setting.

Reply:
Good point. At the moment noreply address is defined in several files. Added a separate JIRA ticket to have only one method to define noreply address and to have environment variable to override it: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1451